edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -885,8 +907,91 @@ def test_update_exam_attempt(self, action, expected_status, mock_update_attempt_
         self.assertEqual(response.status_code, 200)
         mock_update_attempt_status.assert_called_once_with(attempt.id, expected_status)
 
+    def test_get_all_exam_attempts(self):
```

Comment:
I think it might be easier to read if this is separated out into multiple unit tests. The type of test being done here also seems like it belongs in the test_api.py file (because this has to deal with the functionality of the api function you wrote). You should still have view tests, but you can mock what's being returned by the api function to limit the test to the view itself.

Reply:
Noted. I just got my head wrapped around what you're saying here. I'll make a new View and change all of the semantics accordingly first, then I'll implement this.